pkg/operator/ceph/csi/spec.go

```diff
@@ -105,12 +107,13 @@ var (
 // manually challenging.
 var (
 	// image names
-	DefaultCSIPluginImage   = "quay.io/cephcsi/cephcsi:v3.2.1"
-	DefaultRegistrarImage   = "k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.0.1"
-	DefaultProvisionerImage = "k8s.gcr.io/sig-storage/csi-provisioner:v2.0.4"
-	DefaultAttacherImage    = "k8s.gcr.io/sig-storage/csi-attacher:v3.0.2"
-	DefaultSnapshotterImage = "k8s.gcr.io/sig-storage/csi-snapshotter:v3.0.2"
-	DefaultResizerImage     = "k8s.gcr.io/sig-storage/csi-resizer:v1.0.1"
+	DefaultCSIPluginImage         = "quay.io/cephcsi/cephcsi:v3.2.1"
```

Comment:
This will conflict with csi 3.3.0 in #7637. How about we merge the other one first since this depends on it anyway?